File: go/3_math/9_faktorial.go
package main

import (
	"fmt"
)

func Faktorial(N int) int {
	if N == 1 {
		return 1
	} else {
		return N * Faktorial(N-1)
	}
}

func main() {
	fmt.Println(Faktorial(7))  // 5040
	fmt.Println(Faktorial(8))  // 40320
	fmt.Println(Faktorial(9))  // 362880
	fmt.Println(Faktorial(10)) // 3628800
	fmt.Println(Faktorial(12)) // 479001600
}

File: python/3-math/9-faktorial.py
def faktorial(n):
  if n == 1:
    return 1
  else:
    return n * faktorial(n - 1)

print(faktorial(7)) # 5040
print(faktorial(8)) # 40320
print(faktorial(9)) # 362880
print(faktorial(10)) # 3628800
print(faktorial(12)) # 479001600